.github/workflows/ci.yml

```diff
@@ -185,7 +191,6 @@ jobs:
           name: lint-log
           path: plz-out/log
 
-
```

Comment:
**Note**: this change came from the autoformatter, but unless we have a serious reason to keep 2 empty lines here, I would let it be merged.